jobs/eam-integrations/scripts/workday_everfi/api/Everfi/everfi.py

```diff
@@ -142,7 +142,10 @@ def build_comparison_string(rec, locs, loc_map_table, hire_dates):
             if rec.get('relationships','').get('category_labels',''):
                 if len(rec.get('relationships','').get('category_labels','').get('data',''))>0:
                     hire_date_id = rec.get('relationships','').get('category_labels','').get('data','')[0].get('id')
-                    hire_date = hire_dates_inv[hire_date_id]
+                    if hire_dates_inv.get(hire_date_id):
+                        hire_date = hire_dates_inv[hire_date_id]
+                    else:
+                        hire_date = ''
```

Comment:
This can be simplified to:
```suggestion
                    hire_date = hire_dates_inv.get(hire_date_id, '')
```